app/src/main/res/layout/screen_home.xml

```diff
@@ -31,13 +31,9 @@
 
     <Button
       android:id="@+id/helpButton"
-      android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
+      style="@style/Clinic.V2.ToolbarButton"
       android:layout_gravity="end"
-      android:background="@drawable/background_rounded_clearbutton"
-      android:foreground="?selectableItemBackground"
       android:text="@string/home_help"
-      android:textAllCaps="true"
```

Comment:
This attribute got lost. Can we add this to the style too?

Reply:
This is already a part of the text style. It was removed intentionally.